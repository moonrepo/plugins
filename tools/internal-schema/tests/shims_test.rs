use proto_pdk_test_utils::*;
use starbase_sandbox::locate_fixture;

mod schema_tool {
    use super::*;

    #[cfg(not(windows))]
    generate_shims_test!(
        "schema-test",
        ["schema-test"],
        Some(locate_fixture("schemas").join("base.toml"))
    );
}
