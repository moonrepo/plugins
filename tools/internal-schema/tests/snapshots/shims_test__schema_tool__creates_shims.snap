---
source: tools/internal-schema/tests/shims_test.rs
expression: "std :: fs ::\nread_to_string(sandbox.path().join(\".proto/shims/registry.json\")).unwrap()"
---
{
  "schema-test": {}
}
